opengrok-tools/src/main/python/opengrok_tools/utils/mirror.py

```diff
@@ -196,13 +198,13 @@ def get_project_properties(project_config, project_name, hookdir):
         use_proxy, ignored_repos
 
 
-def mirror_project(config, project_name, check_incoming, uri,
+def mirror_project(config, project_name, check_changes, uri,
```

Comment:
the function is getting too long. refactoring needed.

Reply:
agreed

Reply:
pls file a new issue then or maybe even do it now

Reply:
https://github.com/oracle/opengrok/issues/2849